cpp/tests/test_odesecir.cpp

```diff
@@ -720,6 +720,167 @@ TEST(Secir, getInfectionsRelative)
               (100. + 50. + 25.) / (10'000 + 20'000 + 40'000));
 }
 
+TEST(Secir, get_reproduction_number)
+{
+    size_t num_groups = 3;
+    mio::osecir::Model model((int)num_groups);
+
+    mio::ContactMatrixGroup& contact_matrix = model.parameters.get<mio::osecir::ContactPatterns>();
+    contact_matrix[0]                       = mio::ContactMatrix(Eigen::MatrixXd::Constant(3, 3, 10));
+
+    model.parameters.set<mio::osecir::StartDay>(60);
+    model.parameters.set<mio::osecir::Seasonality>(0.2);
+
+    //total population of 10.000
+    model.populations[{mio::AgeGroup(0), mio::osecir::InfectionState::Susceptible}]        = 3000;
```

Comment:
Just to counteract possible errors: I would also set the Confirmed Compartments to 0